### PR TITLE
added netstandard/netstandardapp to known frameworks

### PR DESCRIPTION
**Pull request by xavierdecoster**

### CODE REVIEW

File: src/Core/Utility/VersionUtility.cs [hunk 1]
```diff
@@ -664,6 +678,27 @@ public static string GetShortFrameworkName(FrameworkName frameworkName)
                 // only show version part if it's > 0.0.0.0
                 if (frameworkName.Version > new Version())
                 {
+                    if (frameworkName.Identifier.Equals(NetStandardAppFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetStandardFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetPlatformFrameworkIdentifier, StringComparison.OrdinalIgnoreCase))
+                    {
+                        if (frameworkName.Version.Major > 9
```
Comment (emgarten) [inline src/Core/Utility/VersionUtility.cs]: Shouldn't it just always call GetDecimalVersionString here? Or is the format different?

Comment (xavierdecoster) [inline src/Core/Utility/VersionUtility.cs]: Yep, fixed in https://github.com/NuGet/NuGet2/commit/885e64d38bf4d070309049e0724e1a38b0878d8e

**Comment by emgarten** [comment]
It looks like the method to write the name and GetDecimalVersionString are both checking for digits > 9. This could be combined and cleaned up. 

The change to just add these new frameworks in without compat rules looks safe to me since it will not affect any other frameworks. :shipit:

**Comment by xavierdecoster** [comment]
@emgarten there's indeed a duplicate check there somewhere. Turns out that the first check is required to avoid applying the decimal notation in some cases. Perhaps avoid changing that part in  this PR to reduce impact and keep this change's footprint small?

**Comment by emgarten** [comment]
Sure, and just moving all those > 9 checks to a helper method would clean it up.

**Comment by xavierdecoster** [comment]
@emgarten addressed with https://github.com/NuGet/NuGet2/commit/d62d9829ab19f59563db47f477173d8457a7b63d

Is this change good to go? @yishaigalatzer

**Comment by xavierdecoster** [comment]
bump

**Comment by ericstj** [comment]
@yishaigalatzer any ETA on when this is getting merged?

**Comment by yishaigalatzer** [comment]
This wouldn't make them work on nuget 2, just makes the names recognized. It used mostly for the server.

We also have no immediate plans to release this

**Comment by ericstj** [comment]
I believe this would be required for nuget pack from NuSpec to behave correctly.  Nuget pack is still using nuget.core code.

**Comment by yishaigalatzer** [comment]
Not so much, we are porting it out as we speak

Sent from Outlook Mobilehttp://aka.ms/xp9y6l

On Fri, Feb 12, 2016 at 11:19 AM -0800, "Eric StJohn" <notifications@github.com<mailto:notifications@github.com>> wrote:

I believe this would be required for nuget pack from NuSpec to behave correctly. Nuget pack is still using nuget.core code.

##

**Comment by ericstj** [comment]
Good, that will also be an option for us.  For now I have to use my fork of the pack code to make this work.

**Pull request merged**